packages/desktop-client/src/components/common/Button2.tsx

```diff
@@ -134,13 +137,24 @@ type ButtonProps = ComponentPropsWithoutRef<typeof ReactAriaButton> & {
   variant?: ButtonVariant;
   bounce?: boolean;
   children?: ReactNode;
+  focused?: boolean;
 };
 
 type ButtonVariant = 'normal' | 'primary' | 'bare' | 'menu' | 'menuSelected';
 
 export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
-  (props, ref) => {
-    const { children, variant = 'normal', bounce = true, ...restProps } = props;
+  (props, buttonRef) => {
+    const {
+      children,
+      variant = 'normal',
+      bounce = true,
+      focused = false,
+      ...restProps
+    } = props;
+
+    const ref = useRef<HTMLButtonElement>(null);
+    useProperFocus(ref, focused);
```

Comment:
We should use this hook on the `TransactionsTable` and just pass the ref to the Add and Cancel button instead of doing it directly in the `Button` component.

Reply:
Changed the source code to avoid messing with Button2, can you check? Thank you